src/consistency.rs

```diff
@@ -145,6 +146,32 @@ impl FromCursor for Consistency {
     }
 }
 
+impl FromStr for Consistency {
```

Comment:
I want to dynamically change the consistency from a .env file. This makes it a lot easier.